elitzur-core/src/main/scala/com/spotify/elitzur/validators/Validator.scala

```diff
@@ -271,47 +273,52 @@ object Validator extends Serializable {
         if (!accessor.validator.isInstanceOf[IgnoreValidator[_]]) {
           val name = new JStringBuilder(path.length + accessor.label.length)
             .append(path).append(accessor.label).toString
-          val o = if (accessor.validator.isInstanceOf[FieldValidator[_]]) {
+          val c = config.fieldConfig(name)
+          val (o, validationType) = if (accessor.validator.isInstanceOf[FieldValidator[_]]) {
             val v = accessor.validator.validateRecord(Unvalidated(accessor.value))
             val validationType = accessor.validator.asInstanceOf[FieldValidator[_]].validationType
+            (v, Some(validationType))
+          } else {
 
```

Comment:
Very much a nitpick but this leaves a weird line break here that looks off stylewise